DEVELOPMENT-224: Разработка печатной платы Velvet Controller

SkippiDow opened the pull request:
# Task
https://tracker.yandex.ru/DEVELOPMENT-224

SkippiDow committed:
Актуальный проект. Степ. Актуальный ПДФ схемы

SkippiDow committed:
Добавлен altium bom, в ПП и схему добавлен другой разъём для uSIM

SkippiDow committed:
Добавлен BOM схемы

Kvenecusghost commented:
1. Перепутаны RXTX в GSM
2. Перепутаны RXTX в RFID
3. Может цепи светодиодов называть DOx, а не DIx?

Kvenecusghost reviewed (changes requested):
Нужны поправки в схему

Kvenecusghost commented:
Добавить тестовые пады на все цифровые провода.
Добавить логотип КазАТУ на печатной плате
Разместить рядом с логотипом название и версию: Velvet Controller v2.0
Разместить поле для шести-символьного серийного номера: S/N: XX-XX-XX. Вместо XX необходимо разместить белые прямоугольник для нанесения серийного номера маркером

Kvenecusghost committed:
Подогнал структурную схему и добавил временные файлы в gitignore

Kvenecusghost commented:
Думаю между DVDD и AVDD микросхемы DA1a не помешает ферритовая бусина, чтобы отфильтровать высокочастотные помехи от цифровых сигналов.

Commit:
"Добавить тестовые пады на все цифровые провода.
Добавить логотип КазАТУ на печатной плате
Разместить рядом с логотипом название и версию: Velvet Controller v2.0
Разместить поле для шести-символьного серийного номера: S/N: XX-XX-XX. Вместо XX необходимо разместить белые прямоугольник для нанесения серийного номера маркером"

SkippiDow commented:
проверяй

Commit:
Подписал назначения портов на микроконтроллере. Обновил PDF схемы

Commit:
Добавлена спецификация работы STM32

SkippiDow commented:
Добавил пояснительные надписи на УГО STM32. Добавил спецификацию к микроконтроллеру

Kvenecusghost reviewed (approved):
Принято

Kvenecusghost committed:
Merge branch 'master' into user/ksuvorin/DEVELOPMENT-224
